### PR TITLE
DDT-485 Get booking with carrierbookingrequestreference test

### PR DESCRIPTION
**Pull request by pimg**
resolves DDT-485

### CODE REVIEW

**Review by douradofrazer — DISMISSED** [review]

File: src/test/java/org/dcsa/api_validator/bkg/v1/bookings/GetBookingTest.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+public class GetBookingTest {
+
+	@Test
+	public void testGetValidBookingTest() {
+
+		given()
+			.auth()
+			.oauth2(Configuration.accessToken)
+		.when()
+			.pathParam("carrierBookingRequestReference", "ef223019-ff16-4870-be69-9dbaaaae9b11")
```
Comment (Abdiiir) [inline src/test/java/org/dcsa/api_validator/bkg/v1/bookings/GetBookingTest.java:20]: We should consider making our tests less dependent on the test data?

File: src/test/java/org/dcsa/api_validator/bkg/v1/bookings/GetBookingTest.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+			.body(JSON_SCHEMA_VALIDATOR);
+	}
+
+	@Test
+	public void testGetBookingSomeEmptyDeepObjects() {
+		given()
+			.auth()
+			.oauth2(Configuration.accessToken)
+		.when()
+			.pathParam("carrierBookingRequestReference", "CARRIER_BOOKING_REQUEST_REFERENCE_01")
```
Comment (Abdiiir) [inline src/test/java/org/dcsa/api_validator/bkg/v1/bookings/GetBookingTest.java:35]: same applies here.

**Review by Abdiiir — DISMISSED** [review]

File: src/test/java/org/dcsa/api_validator/bkg/v1/bookings/GetBookingTest.java (hunk 1 above)
Comment (pimg) [reply src/test/java/org/dcsa/api_validator/bkg/v1/bookings/GetBookingTest.java]: changed the tests to be more dynamic

File: src/test/java/org/dcsa/api_validator/bkg/v1/bookings/GetBookingTest.java (hunk 2 above)
Comment (pimg) [reply src/test/java/org/dcsa/api_validator/bkg/v1/bookings/GetBookingTest.java]: changed the tests to be more dynamic

**Review by Abdiiir — APPROVED** [review]

File: src/test/java/org/dcsa/api_validator/bkg/v1/bookings/GetBookingTest.java [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        carrierBookingRequestReference ->
+            given()
+                .auth()
+                .oauth2(Configuration.accessToken)
+                .when()
+                .pathParam("carrierBookingRequestReference", carrierBookingRequestReference)
+                .get(Configuration.ROOT_URI + BOOKING_CARRIERBOOKINGREQUESTREFERENCE_PATH)
+                .then()
+                .assertThat()
+                .header("API-Version", "1.0.0")
```
Comment (Abdiiir) [inline src/test/java/org/dcsa/api_validator/bkg/v1/bookings/GetBookingTest.java:51]: We should check that all elements in retrieved response is the correct carrierBookingRequestReference

` .body("carrierBookingRequestReference", everyItem(equalTo(carrierBookingRequestReference)))`

Which I can see is not the case?

**Review by Abdiiir — CHANGES_REQUESTED** [review]
We should check that all elements in retrieved response is the correct carrierBookingRequestReference

**Pull request merged**